React/Modules/RCTAlertManager.m

```diff
@@ -30,6 +30,7 @@ @interface RCTAlertManager()
 @implementation RCTAlertManager
 {
   NSHashTable *_alertControllers;
+  UIWindow *_window;
```

Comment:
Retaining anything in ViewManager is very bad idea in general (and especially for UIKit things).